feat: added Texture Options to spritesheet loader

sam007mac opened the pull request:
<!--
Thank you for your pull request!

Bug fixes and new features should include tests and possibly benchmarks.

Before submitting please read:

Contributors guide: https://github.com/pixijs/pixijs/blob/dev/.github/CONTRIBUTING.md
Code of Conduct: https://github.com/pixijs/pixijs/blob/dev/.github/CODE_OF_CONDUCT.md
-->

##### Description of change
<!-- Provide a description of the change below this comment. -->
I've added the ability to set TextureOptions on sprites loaded via a spritesheet, as outlined in this Feature Request: #11148 

##### Pre-Merge Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)

I was having difficulty with the test suite but I tested my changes locally and it functioned as intended with normal and multipacked spritesheets.

GoodBoyDigital commented on src/spritesheet/spritesheetAsset.ts:
```diff
@@ -148,7 +158,16 @@ export const spritesheetAsset = {
-                const assets = await loader.load<Texture>([imagePath]);
+                let assets;
+
+                if (textureOptions)
```
I don't think we even need the `if (textureOptions)` check here as textureOptions can be null:

replacing the original line
`assets = await loader.load<Texture>([imagePath]);`

with
`assets = await loader.load<Texture>([{ src: imagePath, data: textureOptions}]);`

should work!

sam007mac replied:
It looks like that check is still needed, removing it caused the SVG spritesheet test to fail. 
I could modify the SVG script to fix that issue but I don't know if that's in the scope of this PR, what are your thoughts?

GoodBoyDigital reviewed (commented):
thanks so much for this @sam007mac ! tiny tweak requested!

bigtimebuddy reviewed (commented):
A test here would be useful to make sure the options are actually set.

sam007mac commented:
I've added a test for checking the texture options.
I've also fixed the issue causing loadSVG to fail because of my change but I don't know if that should be under a separate PR or issue report, it's only partly related.

bigtimebuddy reviewed (approved):
Good fix! Thank you for the test.

GoodBoyDigital reviewed (approved):
thanks @sam007mac 👍
